package.json

```diff
@@ -75,10 +75,6 @@
       "no-throw-literal": [
         "error"
       ],
-      "import/extensions": [
```

Comment:
Had to remove this rule since the node classes use `three/nodes` in their imports.